### ISSUE TITLE
Globaleaks not working after update

### ISSUE THREAD

**Issue by a2ZxgdqHeSs8F51cIyuK** [globaleaks/GlobaLeaks#3739]
### What version of GlobaLeaks are you using?

GlobaLeaks version: 4.13.16 (from 4.13.14)
Debian 12.2

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Windows, Linux

### Describe the issue

After update Globaleaks service starts and reports that it is running, however the portal is no longer reachable.
Neither TOR nor HTTPS are reachable, while they used to be.

Globaleaks.log reports this error continuously:
```
2023-11-02 11:10:12+0000 [-] Starting factory <twisted.mail.smtp.ESMTPSenderFactory object at 0x7f8f7416dd90>
2023-11-02 11:10:12+0000 [-] [E] [1] SMTP connection failed (Exception: Connection was refused by other side: 111: Connection refused.)
2023-11-02 11:10:12+0000 [-] Stopping factory <twisted.mail.smtp.ESMTPSenderFactory object at 0x7f8f7416dd90>
```
Right after the service is started it reports this once:
```
2023-11-02 11:13:18+0000 [-] twistd 22.4.0 (/usr/bin/python3 3.11.2) starting up.
2023-11-02 11:13:18+0000 [-] reactor class: twisted.internet.epollreactor.EPollReactor.
2023-11-02 11:13:20+0000 [-] [E] Found an already initialized database version: 66
2023-11-02 11:13:20+0000 [-] Starting factory <Site object at 0x7f0e2daf5c10>
2023-11-02 11:13:20+0000 [-] [E] Failed to initialize Tor connection; error: Tor control port not open on /var/globaleaks/tmp/tor_control; waiting for Tor to become available
2023-11-02 11:13:20+0000 [-] GlobaLeaks is now running and accessible at the following urls:
2023-11-02 11:13:20+0000 [-] - [Tor]:  [REDACTED]
2023-11-02 11:13:20+0000 [-] Starting factory _HTTP11ClientFactory(<function HTTPConnectionPool._newConnection.<locals>.quiescentCallback at 0x7f0e2ac0a980>, <twisted.internet.endpoints._WrapperEndpoint object at 0x7f0e2ac3c310>)
2023-11-02 11:13:20+0000 [-] Stopping factory _HTTP11ClientFactory(<function HTTPConnectionPool._newConnection.<locals>.quiescentCallback at 0x7f0e2ac0a980>, <twisted.internet.endpoints._WrapperEndpoint object at 0x7f0e2ac3c310>)
2023-11-02 11:13:20+0000 [-] Starting factory <twisted.mail.smtp.ESMTPSenderFactory object at 0x7f0e2adb1490>
2023-11-02 11:13:20+0000 [-] [E] [1] SMTP connection failed (Exception: Connection was refused by other side: 111: Connection refused.)
2023-11-02 11:13:20+0000 [-] Stopping factory <twisted.mail.smtp.ESMTPSenderFactory object at 0x7f0e2adb1490>
2023-11-02 11:13:21+0000 [-] [E] Job ExitNodesRefresh died with runtime -1.0000 [low: -1.0000, high: -1.0000]
2023-11-02 11:13:21+0000 [-] Traceback (most recent call last):
2023-11-02 11:13:21+0000 [-]   File "/usr/lib/python3/dist-packages/globaleaks/jobs/job.py", line 49, in run
2023-11-02 11:13:21+0000 [-]     yield self.operation()
2023-11-02 11:13:21+0000 [-] twisted.internet.error.ConnectionRefusedError: Connection was refused by other side: 111: Connection refused.
2023-11-02 11:13:21+0000 [-] [E] Unhandled exception raised:
2023-11-02 11:13:21+0000 [-] [E] twisted.internet.error.ConnectionRefusedError Connection was refused by other side\n\nTraceback (most recent call last):\n\n  File "/usr/lib/python3/dist-packages/globaleaks/jobs/job.py", line 49, in run\n    yield self.operation()\n\ntwisted.internet.error.ConnectionRefusedError: Connection was refused by other side: 111: Connection refused.\n
```
I suspect the update might have disabled the HTTPS access but the TOR access doesn't work either, so I can't enter and check/reactivate it.

**Comment by evilaliv3** [comment]
Hello, thank you for your question.

Actually the mentioned updates did not change aspects related to networking.

Would you please verify that the application is running? We got reported that in some circumstances globaleaks was not restarting after the update.

If it is running, are you able to access the system on local host either on port 443 or 8443?

**Comment by a2ZxgdqHeSs8F51cIyuK** [comment]
The service reports that it is running. I can stop, start or restart it and the service responds to those commands normally. It stays up if it's running.
Telnet to localhost port 443 reports connection refused.
Telnet to localhost port 8443 connects.
Trying to do the same from a remote machine instead fails on both ports.
I don't have localhost browser access, it's on a CLI only server.

**Comment by evilaliv3** [comment]
GlobaLeaks exposes port 8443 on port 443 via the system firewall (iptables).

I invite you to check if something is wrong with your system firewall.

**Comment by a2ZxgdqHeSs8F51cIyuK** [comment]
I have tried with the default rules added by Globaleaks, tried explicitly allowing every connection from my IP and also tried with a "wiped" itpables on default allow, doesn't work.
Also I did this test, I stop the globaleaks service, then use the netcat command "nc -l -p 8443" so I am certain there is something listening, then tried to telnet, it works. Did the same for port 443.
So there is nothing blocking that port in the middle.
